src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/CodeActions/CodeActionsHandler.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { RequestType } from 'vscode-languageclient';
+import { RazorDocumentManager } from '../RazorDocumentManager';
+import { RazorDocumentSynchronizer } from '../RazorDocumentSynchronizer';
+import { RazorLanguageFeatureBase } from '../RazorLanguageFeatureBase';
+import { RazorLanguageServerClient } from '../RazorLanguageServerClient';
+import { RazorLanguageServiceClient } from '../RazorLanguageServiceClient';
+import { RazorLogger } from '../RazorLogger';
+import { RazorCodeAction } from '../RPC/RazorCodeAction';
+import { SerializableCodeActionParams } from '../RPC/SerializableCodeActionParams';
+import { convertRangeFromSerializable } from '../RPC/SerializableRange';
+
+export class CodeActionsHandler
+    extends RazorLanguageFeatureBase {
```

Comment:
This feature base is only relevant for features that call `getProjectionAsync`. In your case you don't so I don't think you need all of the ceremony/requirements this base class requires

Reply:
Yep thanks. Removed the need for `getProjectionAsync` but missed removing the base class.